packages/labs/polyfills/scoped-registry.js

```diff
@@ -0,0 +1,330 @@
+/**
```

Comment:
We'll try and get a release of this (along with `template-shadowroot`, which is also needed for `lit-next` out soon, so that this doesn't have to live here long-term.

Reply:
Awesome! 🤗